### PR TITLE
Feature eugenia api teacher

### PR DESCRIPTION
**Pull request by emusat2021**
API Teacher

### CODE REVIEW

File: RubyRangersLMS_API/TeachersFakeData.cs
Comment (ghost) [inline RubyRangersLMS_API/TeachersFakeData.cs:1]: Jag tror inte att det är bra att lägga in falska data i Developer Bransh. Det är min åsikt.

File: RubyRangersLMS_API/ViewModels/TeacherCreateModel.cs
Comment (ghost) [inline RubyRangersLMS_API/ViewModels/TeacherCreateModel.cs:1]: Det kunde skapas en klass med samma properties och kalla den t.ex teacherModel eller TeacherView model och använda den var den behövs

File: RubyRangersLMS_API/ViewModels/TeacherDeleteModel.cs
Comment (ghost) [inline RubyRangersLMS_API/ViewModels/TeacherDeleteModel.cs:1]: det kunde skapas en klass med samma properties och kalla den t.ex teacherModel eller TeacherView model och använda den var den behövs

File: RubyRangersLMS_API/ViewModels/TeacherViewModel.cs
Comment (ghost) [inline RubyRangersLMS_API/ViewModels/TeacherViewModel.cs:1]: det kunde skapas en klass med samma properties och kalla den t.ex teacherModel eller TeacherView model och använda den var den behövs

File: RubyRangersLMS_API/Entities/Teacher.cs
Comment (ghost) [inline RubyRangersLMS_API/Entities/Teacher.cs:1]: Varför har du tagit bort IdentityUser arv från klassen ? Ska vi inte använda den?

**Review by ghost — COMMENTED** [review]
Läsa på mina kommentarer, sen vi kan diskutera dem om du vill?

File: RubyRangersLMS_API/Entities/Teacher.cs
Comment (emusat2021) [reply RubyRangersLMS_API/Entities/Teacher.cs:1]: Please tell me the line number in "RubyRangersLMS_API/Entities/Teacher.cs" where you say that I removed  inheritance of  "IdentityUser"!

File: RubyRangersLMS_API/TeachersFakeData.cs
Comment (emusat2021) [reply RubyRangersLMS_API/TeachersFakeData.cs:1]: I removed the fake file

File: RubyRangersLMS_API/Entities/Teacher.cs
Comment (ghost) [reply RubyRangersLMS_API/Entities/Teacher.cs:1]: I thank I made a mistake, I check your branch and IdentityUser is still there. you can ignore this comment

File: RubyRangersLMS_API/ViewModels/TeacherCreateModel.cs
Comment (emusat2021) [reply RubyRangersLMS_API/ViewModels/TeacherCreateModel.cs:1]: I deliberately chose to create different classes, one for each endpoint, to have more control over returned data structure in each endpoint.
Of course one can improve the code and have only one class but this also works.
Your suggestion is a nice to have feature not must have.

File: RubyRangersLMS_API/ViewModels/TeacherDeleteModel.cs
Comment (emusat2021) [reply RubyRangersLMS_API/ViewModels/TeacherDeleteModel.cs:1]: I deliberately chose to create different classes, one for each endpoint, to have more control over returned data structure in each endpoint.
Of course one can improve the code and have only one class but this also works.
Your suggestion is a nice to have feature not must have.

File: RubyRangersLMS_API/ViewModels/TeacherViewModel.cs
Comment (emusat2021) [reply RubyRangersLMS_API/ViewModels/TeacherViewModel.cs:1]: I deliberately chose to create different classes, one for each endpoint, to have more control over returned data structure in each endpoint.
Of course one can improve the code and have only one class but this also works.
Your suggestion is a nice to have feature not must have.

**Review by ghost — APPROVED** [review]

**Pull request merged**